coordinator/src/substrate/mod.rs

```diff
@@ -568,25 +568,25 @@ async fn handle_new_blocks<D: Db, Pro: Processors>(
     )
     .await?;
     *next_block += 1;
-    db.set_next_block(*next_block);
+    let mut txn = db.txn(); // TODO: this line belongs to a removed method, remove SubstrateDb
```

Comment:
This TODO doesn't seem still needed.

Reply:
Ahh yeah, sorry I can remove that. I was going to ask you about that but then I just made the call. There was a method that committed the transaction right away and I was thinking about making it a helper but decided against it